src/AcmSearchbar/AcmSearchbar.css

```diff
@@ -162,6 +162,9 @@
     cursor: pointer;
     background: var(--pf-global--BackgroundColor--200);
 }
+.react-tags__suggestions li.is-active {
+    background: var(--pf-global--BackgroundColor--200);
+}
```

Comment:
Is this the fix for the arrows?

Reply:
Yep, just missing the background-color on active state